### ISSUE TITLE
Possible to add this to variants: {} for hover, responsive and group-hover utils?

### ISSUE THREAD

**Issue by frontedbe** [joshwaaaah/tailwind-underline-utilities#5]
```
variants: {
        underlineColor: ['responsive', 'hover', 'group-hover']
    },
    
plugins: [
        require('tailwind-underline-utils')
    ],
```

Doesn't seem to work when I try to group-hover:underline-black on an item.

**Comment by joshwaaaah** [comment]
Hi @mdominguez ,

Thanks for the feedback. I'll take a look into this when I get time and update with a fix :)

**Comment by joshwaaaah** [comment]
@mdominguez this should be solved now in the latest version (1.1.3)

Thanks for raising and let me know if you have any other issues :)
